transform/mattermost-analytics/models/staging/mm_calls_test_go/base/base_mm_calls_test_go__tracks.sql

```diff
@@ -0,0 +1,12 @@
+{%- set include_columns = ["call_id", "actual_user_id", "participants", "server_version", "duration", 
```

Comment:
`tracks` table is meant to keep information shared between all events.

After checking the code in [calls server](https://github.com/mattermost/mattermost-plugin-calls/blob/main/server/telemetry.go#L152) and [calls front-end](https://github.com/mattermost/mattermost-plugin-calls/blob/main/webapp/src/actions.ts#L220), the following columns appearing on all events:
- `SERVER_VERSION`
- `PLUGIN_BUILD`
- `PLUGIN_VERSION`
- `ACTUAL_USER_ID`

`SOURCE` and `CLIENT_TYPE` also seem to be there for all user events.

`duration`, `screen_duration` and `call_id` do seem to be available only for a small subset of the events. If these fields are needed, then a separate union of the specific event tables can be used. This will also help with performance as (a) tracks table will not have columns with a lot of null values and (b) all events with a `call_id` will be available in a separate table, reducing the processing time.

Based on the above, the list of extra columns to be included must be adjusted.

Reply:
Updated tracks table, removed the `duration`, `screen_duration` and `call_id` columns, and `source` and `client_type` are only present in `USER` tables, they aren't in the main tables (`CALL_STARTED`, `CALL_ENDED`) etc. these will also get the same treatment (if they are required, right now they aren't) in a separate table.